internal/installercache/installercache_test.go

```diff
@@ -119,19 +145,162 @@ var _ = Describe("installer cache", func() {
 		}
 		Expect(l.Path).ShouldNot(BeEmpty())
 
-		time.Sleep(1 * time.Second)
+		time.Sleep(10 * time.Millisecond)
```

Comment:
Why is this sleep necessary? Can you add a comment explaining it if it really is needed?

Reply:
I'll add a comment, the general idea was to add a slight simulation of a release being held by the caller, but I didn't want this simulation to take a lot of time, just enough to perhaps force some waiting during the tests.

Reply:
You are right, I just tested this function again and we don't need this for the assertions we are performing here.

I think I was attempting to simulate a small amount of latency between fetching a release using `installers.Get(...)` and calling `Cleanup(...)` on the release. In practice however, this is not needed for this simple, non parallel test.